common/beerocks/bcl/include/bcl/network/network_utils.h

```diff
@@ -136,4 +136,11 @@ inline el::base::MessageBuilder &operator<<(el::base::MessageBuilder &log, const
     return log << beerocks::net::network_utils::mac_to_string(addr);
 }
 
+inline bool operator<(const sMacAddr &lmac, const sMacAddr &rmac)
+{
+    auto s1 = beerocks::net::network_utils::mac_to_string(lmac);
+    auto s2 = beerocks::net::network_utils::mac_to_string(rmac);
+    return (s1.compare(s2));
+}
```

Comment:
This is only needed because you're using a std::set or std::map somewhere, which you shouldn't do.